Sources/Commands/SwiftBuildTool.swift

```diff
@@ -37,11 +37,11 @@ private enum Mode: Argument, Equatable, CustomStringConvertible {
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
-        case "--configuration", "--conf", "-c":
+        case "--configuration", "--config":
```

Comment:
Ok, I've added it back.  There is perhaps a broader conversation to be had about whether we in general support old-style single-dash/single-char short forms or just the double-dash/whole-word forms.  I've looked at various tools for example and they seem all over the map about this, and not just the ancient ones either.

But we certainly don't want to break anything, so I've added it back.